scripts/zones/Ceizak_Battlegrounds/IDs.lua

```diff
@@ -25,6 +25,8 @@ zones[xi.zone.CEIZAK_BATTLEGROUNDS] =
         ARTIFACT_TERMINAL_VOLUME      = 7615, -- Your artifact has been charged to its terminal volume of kinetic units.
         SURPLUS_LOST_TO_AETHER        = 7616, -- A surplus of <number> kinetic unit[/s] has been lost to the aether.
         HOMEPOINT_SET                 = 7798, -- Home point set!
+        MONSTER_APPEAR                = 8021, -- A Monster has appeared.
```

Comment:
Our client update tooling relies on these comments being _exactly_ what's in the game client. This string needs to be: A monster appears!

The next one seems to be correct: The spring has returned to your step.